client/src/components/table/Table.js

```diff
@@ -2,21 +2,23 @@ import React, { useState, useEffect } from 'react';
 import { AgGridReact } from 'ag-grid-react';
 
 import columnDefs from '../../utils/tableHelpers';
-import mockMemberData from '../../utils/mockMemberData';
+import { getMembers } from '../../utils/apiWrapper';
 import '../../css/Table.css';
 import 'ag-grid-community/dist/styles/ag-grid.css';
 import 'ag-grid-community/dist/styles/ag-theme-alpine.css';
 
 const Table = () => {
-  const [members] = useState(mockMemberData);
-  const getMembers = async () => {
-    // TODO: replace with getMembers() api call
-  };
-
+  const [members, setMembers] = useState([]);
   useEffect(() => {
-    getMembers();
+    const getAllMembers = async () => {
+      const allMembers = await getMembers()
+      setMembers(allMembers)
```

Comment:
ur super close!! the axios response object is actually formatted like this
![Screen Shot 2021-01-22 at 8 58 02 PM](https://user-images.githubusercontent.com/32147571/105568342-32e13280-5cfe-11eb-9dff-7a45e629a4da.png)

So you would need to access the actual member data like this: `setMembers(allMembers.data.result)`